Test/Mono.Cecil.Tests/BaseTestFixture.cs

```diff
@@ -119,19 +119,21 @@ abstract class TestCase {
 		public readonly bool ReadOnly;
 		public readonly Type SymbolReaderProvider;
 		public readonly Type SymbolWriterProvider;
+		public readonly BaseAssemblyResolver AssemblyResolver;
```

Comment:
That's what I did first. But IAssemblyResolver doesn't support AddSearchDirectory method which is called by TestRunner.GetModule.